test/jdk/java/awt/print/PrinterJob/SecurityDialogTest.java

```diff
@@ -64,7 +66,7 @@ public static void main(String[] args) throws Exception {
             throw new RuntimeException("Printer not configured or available.");
         }
 
-        PassFailJFrame passFailJFrame = new PassFailJFrame.Builder()
+        PassFailJFrame passFailJFrame =  PassFailJFrame.builder()
```

Comment:
```suggestion
        PassFailJFrame passFailJFrame = PassFailJFrame.builder()
```
Removed redundant double space.